armi/reactor/reactors.py

```diff
@@ -132,6 +132,10 @@ def factory(cs, bp, geom: Optional[geometry.SystemLayoutInput] = None):
 
     with directoryChangers.DirectoryChanger(cs.inputDirectory, dumpOnException=False):
         # always construct the core first (for assembly serial number purposes)
+        if not bp.systemDesigns:
```

Comment:
This isn't checking specifically for a "core" system blueprint...

Reply:
Directly below it is, so if this branch of the code is being executed, a `core` system is being sought.